src/materials/Material.js

```diff
@@ -194,6 +194,7 @@ Material.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		if ( this.normalMap && this.normalMap.isTexture ) {
 
 			data.normalMap = this.normalMap.toJSON( meta ).uuid;
+			if ( this.normalMapType !== TangentSpaceNormalMap ) data.normalMapType = this.normalMapType;
```

Comment:
nuke, would be serialized with the texture, there are possibly materials that extend `Material` that do not use normal maps, like maybe depth. Consider not making the super class aware of the sub classes.